ssl/ssl_lib.c

```diff
@@ -4074,59 +4070,168 @@ int SSL_CTX_set_ct_validation_callback(SSL_CTX *ctx, ct_validation_cb callback,
             TLSEXT_TYPE_signed_certificate_timestamp)) {
         SSLerr(SSL_F_SSL_CTX_SET_CT_VALIDATION_CALLBACK,
                SSL_R_CUSTOM_EXT_HANDLER_ALREADY_INSTALLED);
-        goto err;
+        return 0;
     }
 
     ctx->ct_validation_callback = callback;
     ctx->ct_validation_callback_arg = arg;
-    ret = 1;
-err:
-    return ret;
+    return 1;
+}
+
+int SSL_ct_is_enabled(const SSL *s)
```

Comment:
Should the first impression be "this is a boolean predicate" or "this is a ct interface"?  I don't care.
Would prefer to leave this for Rob & co. to decide (I do expect a follow on PR/MR) that picks up the remaining loose ends.